tests/unit_tests/cuda/grids_grid2_cuda_kernel.cuh

```diff
@@ -0,0 +1,22 @@
+/** Detray library, part of the ACTS project (R&D line)
```

Comment:
I don't understand this header... :frowning: First off, I don't think `.cuh` is a good extension for it. (Currently it does not include any CUDA code, so `.hpp` would be more appropriate.)

But since it declares 2 template functions, but it does not include the implementation of those 2 functions, I'm really not sure what its purpose is... :confused:

Reply:
implementation is in .cu file in the same directory.  I thought it is pretty normal to name it .cuh when it contains declaration of .cu file

Do you know how to put implementation in the same file? AFAIK this separation is the only way.